napari_cellseg3d/models/model_SwinUNetR.py

```diff
@@ -0,0 +1,18 @@
+from monai.networks.nets import SwinUNETR
+
+
+def get_weights_file():
+    return ""
+
+
+def get_net():
+    return SwinUNETR
+
+
+def get_output(model, input):
```

Comment:
Maybe make a function with image size here ? 

 ```suggestion
def get_output(model, input, size):
return SwinUNetR(img_size=utils.get_padding_dim(size),
                in_channels=1,
                out_channels=1,
                feature_size=48,
                use_checkpoint=True,
            )

```